starter-code/src/App.js

```diff
@@ -1,18 +1,78 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import "bulma/css/bulma.css";
+import "./App.css";
+import foods from "./foods.json";
+import FoodBox from "./components/FoodBox";
+import FormAddFoodBox from "./components/FormAddFoodBox";
+import SearchBar from "./components/SearchBar";
+import TodaysFoods from "./components/TodaysFoods";
 
 class App extends Component {
+  state = {
+    isActive: false,
+    foods: foods,
+    todaysFoods: [],
+    totalCal: 0
+  };
+
+  handleClick = e => {
+    this.setState({ isActive: true });
+  };
+
+  addMyFood = v => {
+    const copy = [...this.state.todaysFoods];
+    copy.push(v);
+    const total = copy
+      .map(item => item.calories * item.quantity)
+      .reduce((prev, next) => prev + next);
+    this.setState({ todaysFoods: copy, totalCal: total });
+  };
+
+  addProduct = info => {
+    this.setState({ foods: [...this.state.foods, info], isActive: false });
+  };
+
+  handleChange = input => {
+    const searchValue = input.value;
+    let filtered = foods.filter(f =>
+      f.name.toLowerCase().includes(searchValue.toLowerCase())
+    );
+    this.setState({ foods: filtered });
```

Comment:
for this lab you should not filter directly the state of foods otherwise, you loose the product you created before.